conf/machine/include/imx-base.inc

```diff
@@ -372,6 +379,7 @@ IMX_EXTRA_FIRMWARE:mx8-generic-bsp   = "firmware-imx-8 imx-sc-firmware imx-seco"
 IMX_EXTRA_FIRMWARE:mx8m-generic-bsp  = "firmware-imx-8m"
 IMX_EXTRA_FIRMWARE:mx8x-generic-bsp  = "imx-sc-firmware imx-seco"
 IMX_EXTRA_FIRMWARE:mx8ulp-generic-bsp = "firmware-upower firmware-sentinel"
+IMX_EXTRA_FIRMWARE:mx9-nxp-bsp        = " firmware-imx-8 firmware-sentinel"
```

Comment:
Please, use `mx93` override.

Reply:
FYI, I started to suggest that we use `mx9` to denote the family as we do for `mx8m` and `mx8x`, but I remembered that the naming scheme has changed and `mx8m` is analogous to `mx93` or `mx95`. This is an improvement on the naming since for i.MX 8 we had the ambiguity of an i.MX 8 Series and an i.MX 8 Family.